README.md

```diff
@@ -44,9 +44,9 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
 
 ## Synopsys Security Scan - Polaris
 
-Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline. The extension will download the Synopsys Bridge CLI, execute a scan, and offers post-scan features such as break-the-build quality gates.
 
-Here's an example piepline for Polaris scan using the Synopsys Synopsys Security Scan:
+Here's an example pipeline for Polaris scan using the Synopsys Synopsys Security Scan:
```

Comment:
in polari exa,ple we have used '' in right side of parameter values. and assessment_types are "". is that fine?

Reply:
yes